### PR TITLE
Fixed sonar issues

### PR DESCRIPTION
**Pull request by nilasini**
Fixed sonar issues

### CODE REVIEW

File: components/org.wso2.carbon.identity.sts.mgt/src/main/java/org/wso2/carbon/identity/sts/mgt/internal/IdentitySTSMgtServiceComponent.java [hunk 1]
```diff
@@ -279,10 +282,14 @@ private void loadSecurityScenarios() throws Exception {
...
-                    !scenarioId.equals(SecurityConstants.POLICY_FROM_REG_SCENARIO)) {
+                        !scenarioId.equals(SecurityConstants.POLICY_FROM_REG_SCENARIO)) {
                     Resource scenarioResource = new ResourceImpl();
+                    URL urlResource = bundleContext.getBundle().getResource(scenarioId + "-policy.xml");
+                    if (urlResource == null) {
+                        return;
+                    }
                     scenarioResource.setContentStream(
-                            bundleContext.getBundle().getResource(scenarioId + "-policy.xml").openStream());
+                            urlResource.openStream());
```
Comment (madurangasiriwardena) [inline components/org.wso2.carbon.identity.sts.mgt/src/main/java/org/wso2/carbon/identity/sts/mgt/internal/IdentitySTSMgtServiceComponent.java]: can we get this to one line.

File: components/org.wso2.carbon.identity.sts.passive/src/main/java/org/wso2/carbon/identity/sts/passive/PassiveSTSService.java [hunk 2]
```diff
@@ -227,6 +227,7 @@ private void setReplyToURL(RequestToken request) {
         if(sp == null) {
```
Comment (madurangasiriwardena) [inline components/org.wso2.carbon.identity.sts.passive/src/main/java/org/wso2/carbon/identity/sts/passive/PassiveSTSService.java]: Formatting issue

Comment (nilasini) [reply components/org.wso2.carbon.identity.sts.passive/src/main/java/org/wso2/carbon/identity/sts/passive/PassiveSTSService.java]: fixed with https://github.com/wso2/carbon-identity-framework/pull/1303

File: components/org.wso2.carbon.identity.sts.mgt/src/main/java/org/wso2/carbon/identity/sts/mgt/internal/IdentitySTSMgtServiceComponent.java (hunk 1 above)
Comment (nilasini) [reply components/org.wso2.carbon.identity.sts.mgt/src/main/java/org/wso2/carbon/identity/sts/mgt/internal/IdentitySTSMgtServiceComponent.java]: Fixed with https://github.com/wso2/carbon-identity-framework/pull/1303

Comment (nilasini) [reply components/org.wso2.carbon.identity.sts.mgt/src/main/java/org/wso2/carbon/identity/sts/mgt/internal/IdentitySTSMgtServiceComponent.java]: It has been fixed with https://github.com/wso2-extensions/identity-inbound-auth-sts/pull/63/files#diff-dfe5fbb75fd0b55201fb57346f45b229R289

File: components/org.wso2.carbon.identity.sts.passive/src/main/java/org/wso2/carbon/identity/sts/passive/PassiveSTSService.java (hunk 2 above)
Comment (nilasini) [reply components/org.wso2.carbon.identity.sts.passive/src/main/java/org/wso2/carbon/identity/sts/passive/PassiveSTSService.java]: Had been fixed

**Pull request merged**